From ff92bee9edc0637321cdb5f55eaf587fda3d8082 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Fri, 16 Sep 2016 01:00:24 +0100
Subject: [PATCH 12/12] daemon: Address shadow on EndSessionDialog in composite
 race-condition

We're starting too early and the composite/screen status changes too much
for Gdk to keep up in early startup, resulting in *sometimes* having the
RGBA visual.

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 daemon/budgie-daemon.desktop.in.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/daemon/budgie-daemon.desktop.in.in b/daemon/budgie-daemon.desktop.in.in
index 0133ddd..dd144db 100644
--- a/daemon/budgie-daemon.desktop.in.in
+++ b/daemon/budgie-daemon.desktop.in.in
@@ -9,5 +9,5 @@ NoDisplay=true
 StartupNotify=true
 X-GNOME-AutoRestart=true
 X-GNOME-Autostart-Notify=true
-X-GNOME-Autostart-Phase=Panel
+X-GNOME-Autostart-Phase=Desktop
 OnlyShowIn=Budgie
-- 
2.10.0

